tsconfig.json

```diff
@@ -11,9 +11,9 @@
     "removeComments": false,
     "experimentalDecorators": true,
     "preserveConstEnums": true,
-    "outDir": "./dist/",
+    "outDir": "./dist",
     "sourceMap": true,
-    "declarationDir": "./dist/"
   },
-  "include": ["src/**/*", "test/**/*"]
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "test"]
```

Comment:
Why did you move the test directory from included to excluded?

Reply:
It keeps the build output clean and free of test-related artifacts.
Previously they were included, but for what reason? https://www.npmjs.com/package/@oceanprotocol/lib/v/3.3.0?activeTab=code

Reply:
@jamiehewitt15 what do you think?